utils/get-us-data.R

```diff
@@ -1,78 +1,77 @@
-# Get & reshape JHU data
+# Get & reshape JHU data for deaths or cases
+# 
+# Args:
+# include_national FALSE whether to include US as the sum of states
+# incident_daily TRUE returns incident, FALSE returns cumulative
+# incident_weekly FALSE whether to add a column as weekly sum of daily incidence
+#
 # Source: https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data
-  
-library(magrittr)
 
-# Arguments
-# data = c("cumulative", "daily")
-# anomaly_threshold = minimum death count above which to adjust anomalies (e.g 100)
-# check_adjustmente = boolean, if TRUE prints which states have had data adjusted
+library(magrittr)
+library(dplyr)
+library(tidyr)
+library(readr)
+library(lubridate)
   
-# Deaths data -------------------------------------------------------------
-get_us_deaths <- function(data = "daily"){  
+get_us_data <- function(data = c("deaths", "cases"),
+                        include_national = FALSE,
+                        incident = TRUE, 
+                        incident_weekly = FALSE) {  
+  
```

Comment:
if I was being fussy it would be nice to use match.arg on the data arg and not have the default be something that will break the function/throw a warning.